client/src/components/AddToCartComponent/AddToCartComponent.jsx

```diff
@@ -0,0 +1,27 @@
+// import styles from './AddToCartComponent.module.scss';
+// import React from 'react';
+
+
+// export const AddToCartComponent = () => {
+//     const addToCart = () => {
+//         console.log(`Menge ${quantity} zum Einkaufswagen hinzugefügt!`);
+//     };
+//     return (
+//         <>
+//             <button className={styles.addCartButton} onClick={addToCart}>Add to Cart</button>
+//         </>
+//     )
+// };
+
+
+import styles from './AddToCartComponent.module.scss';
+import React from 'react';
+
+export const AddToCartComponent = ({ quantity, addToCart }) => {
+    return (
+        <>
+            <button className={styles.addCartButton} onClick={addToCart}>Add to Cart</button>
+        </>
+    )
```

Comment:
Kannst du so machen, geht aber auch ein bisschen schöner so:
```suggestion
    return <button className={styles.addCartButton} onClick={addToCart}>Add to Cart</button>
```
So oder so die funktionsweise ist die gleiche :)

Reply:
meinst du wegen der klammern? der hat mir ein fehler gezeigt, wenn ich keine klammer benutze, aber wenn es keinen unterschied macht, lasse ich es einfach so ;)

Reply:
> meinst du wegen der klammern? der hat mir ein fehler gezeigt, wenn ich keine klammer benutze, aber wenn es keinen unterschied macht, lasse ich es einfach so ;)

nicht unbedingt ohne klamern aber ohne Fragment aufjedenfall